ingest/bin/post_process_metadata.py

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/env python3
+
+"""Reformat pandas DataTables for a pathogen build.
+
+Expecting one argument, a NCBI Virus metadata.tsv file
+"""
+# ===== Dependencies
```

Comment:
_non-blocking_

This sort of trivial comment is extraneous. There's a few more below too.

Is there a reason for them that I'm missing?

Reply:
I'll remove the trivial comments, it's a reminder to myself of the different sections of a script but should be removed before merging to main/master.